client/my-sites/plans/current-plan/ecommerce-trial/index.tsx

```diff
@@ -263,10 +253,7 @@ const ECommerceTrialCurrentPlan = () => {
 
 			<div className="e-commerce-trial-current-plan__cta-wrapper">
 				<Button className="e-commerce-trial-current-plan__cta is-primary">
-					{
-						// TODO: translate when final copy is available
-						'Enhance your store and get Commerce'
-					}
+					{ translate( 'Upgrade now' ) }
 				</Button>
```

Comment:
Same again for making the button add the product to the cart, and record an event in Tracks.